src/main/java/org/jboss/logmanager/formatters/PatternFormatter.java

```diff
@@ -19,14 +19,20 @@
 
 package org.jboss.logmanager.formatters;
 
+import java.util.regex.Pattern;
+
 /**
  * A formatter which uses a text pattern to format messages.
  */
 public class PatternFormatter extends MultistepFormatter {
+    private static final Pattern CALLER_CHECK_PATTERN = Pattern.compile(".*(%M|%C|%F|%L|%l).*");
```

Comment:
This pattern will miss items that have field widths, won't it? I think you might want to just query the format steps directly instead maybe.